tasks/app2-2.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const router = require('../tasks/routes/usersWithValidation.js');
+
+const app = express();
```

Comment:
No need to create a separate server for each task from now on :)
Consider just extending the existing functionality or adding a separate endpoint if you need to.